Gridy/View/ImageEditorView.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  ImageEditorView.swift
+//  Gridy
+//
+//  Created by Alexandra Ivanova on 21/10/2020.
+//  Copyright © 2020 Alex Ivanova. All rights reserved.
+//
+
+import UIKit
+
+class ImageEditorView: UIView {
+    private var startButton: UIButton!
+    private var instructionsLabel: UILabel!
+    private var closeButton: UIButton!
+    
+    private var clearView: UIView!
+    private var initialUIImageViewCenter: CGPoint?
+    
+    var imagesBound: [CGRect]!
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    init(image: UIImage) {
+      super.init(frame: .zero)
+      self.clipsToBounds = true
+      self.backgroundColor = UIColor.lightGray
+      self.translatesAutoresizingMaskIntoConstraints = false
+    }
+    
+    func setup(parentView view: UIView) {
+      view.backgroundColor = UIColor.white
+      view.addSubview(self)
+      let safeArea = view.safeAreaLayoutGuide
```

Comment:
I think that `self.safeAreaLayoutGuide` is the same than `parent.safeAreaLayoutGuide`